sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue

```diff
@@ -1,103 +1,151 @@
-        <form>
-          <p>There is already a draft of an application in progress.</p>
-          <p>Please continue your draft application or cancel it.</p>
-        </form>
-      </v-container>
-    </template>
-    <template v-slot:footer>
-      <footer-buttons
-        secondaryLabel="Close"
-        :showPrimaryButton="false"
-        @secondaryClick="dialogClosed"
-      />
-    </template>
-  </modal-dialog-base>
-  <student-page-container>
-    <v-sheet elevation="1" class="mx-auto">
-      <v-container>
-        <formio
-          formName="studentapplication"
-          :data="initialData"
-          @loaded="formLoaded"
-          @customEvent="customEventCallback"
-        ></formio>
-      </v-container>
-    </v-sheet>
-  </student-page-container>
+  <v-form ref="startApplicationForm">
+    <modal-dialog-base
+      title="Application already in progress"
+      dialogType="warning"
+      :showDialog="showDialog"
+      @dialogClosed="closeDialog"
+    >
+      <template v-slot:content>
+        <v-container>
+          <form>
+            <p>
+              There is already an application selected with this program year.
+              Please continue with that application or cancel it to start a new
+              application.
+            </p>
+          </form>
+        </v-container>
+      </template>
+      <template v-slot:footer>
+        <footer-buttons
+          primaryLabel="Close"
+          :showSecondaryButton="false"
+          @primaryClick="closeDialog"
+        />
+      </template>
+    </modal-dialog-base>
+
+    <student-page-container container-class="container-max-width ma-auto">
+      <template #header>
+        <header-navigator
+          title="Applications"
+          subTitle="Start New Application"
+          :routeLocation="{
+            name: StudentRoutesConst.STUDENT_APPLICATION_SUMMARY,
+          }"
+        >
+          <template #buttons>
+            <start-application />
+          </template>
+        </header-navigator>
+      </template>
+      <h2 class="category-header-large primary-color pb-4">
+        Apply for funding
+      </h2>
+      <content-group>
+        <div class="category-header-medium-small pa-2">
+          Financial Aid Application
+        </div>
+        <div class="pb-6 px-2">
+          Apply to see if you are applicable for provincial or federal loans and
+          grant. This is for students attending full-time or part-time
+          post-secondary studies.
+        </div>
+        <div class="category-header-medium-small pb-3 px-2">
+          Select program year
+        </div>
+        <v-select
+          v-model="programYearId"
+          :items="programYearOptions"
+          variant="solo"
+          density="compact"
+          label="Select year"
+          class="px-2 pb-4"
+          :rules="[(v) => checkNullOrEmptyRule(v, 'Year')]"
+          hide-details="auto"
+        ></v-select>
+        <v-btn
+          class="ma-2"
```

Comment:
what is `"ma-2"`?

Reply:
a - applies the spacing for the property in all directions

Reply:
May be I am missing something. I didn't get what all directions means here? just using margin will put margin to all directions right?

Reply:
I didn't get what you mean. You can check Vuetify Spacing helpers here: https://vuetifyjs.com/en/styles/spacing/

Reply:
mm.. got it. so it's the same as bootstrap `"m-2"`